cfme/physical/provider/__init__.py

```diff
@@ -2,12 +2,21 @@
 from widgetastic.utils import Fillable
 
 from cfme.base.ui import Server
+
+from cfme.base.ui import BaseLoggedInPage
+from cfme.utils import version
 from cfme.common.provider import BaseProvider
-from cfme.common.provider_views import PhysicalProviderAddView, PhysicalProvidersView
+from cfme.common.provider_views import (PhysicalProviderAddView,
+                                        PhysicalProviderDetailsView,
+                                        PhysicalProviderEditView)
 from cfme.utils.appliance import Navigatable
 from cfme.utils.appliance.implementations.ui import navigator, CFMENavigateStep
 from cfme.utils.pretty import Pretty
 from cfme.utils.varmeth import variable
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.fixtures import pytest_selenium as sel
```

Comment:
We will almost never accept a new import from pytest_selenium as we're trying to phase it out.

Here you can get import NoSuchElementException from widgetastic.exceptions instead